testing/logging/test_reporting.py

```diff
@@ -1002,3 +1002,49 @@ def pytest_sessionfinish(session, exitstatus):
         assert "sessionstart" in contents
         assert "runtestloop" in contents
         assert "sessionfinish" in contents
+
+def test_log_set_path(testdir):
+    from pathlib2 import Path
```

Comment:
This needs to be `from _pytest.pathlib import Path` so it works for all Python versions we support.

Reply:
Sorry, about that. Previously I already used that way (as you mentioned), just forgot. Soon I will fix.

Reply:
Updated